manifests/install/charts/as-a-second-scheduler/templates/configmap.yaml

```diff
@@ -10,13 +10,10 @@ data:
     leaderElection:
       leaderElect: false
     profiles:
-    - schedulerName: scheduler-plugins-scheduler
+    # user can add as many profiles here. Adding two profiles as an example
+{{- if has "coscheduling" .Values.plugins.enable}}
+    - schedulerName: Coscheduling-plugins-scheduler
```

Comment:
I'd suggest starting with a one-profile to include all plugins:

```yaml
- schedulerName: scheduler-plugins-scheduler
```

and just inject the Helm if directives around the particular plugins.

Otherwise, the test pod has to specify its `schedulerName` differently.